diff --git a/node_modules/svelte-multiselect/MultiSelect.svelte b/node_modules/svelte-multiselect/MultiSelect.svelte
index de22a9a..1eebad8 100644
--- a/node_modules/svelte-multiselect/MultiSelect.svelte
+++ b/node_modules/svelte-multiselect/MultiSelect.svelte
@@ -43,6 +43,8 @@ export let required = false;
 export let autocomplete = `off`;
 export let invalid = false;
 export let sortSelected = false;
+export let closeOptionsIfNoneLeft = false;
+
 if (maxSelect !== null && maxSelect < 1) {
     console.error(`maxSelect must be null or positive integer, got ${maxSelect}`);
 }
@@ -152,6 +154,11 @@ async function handleKeydown(event) {
     if (event.key === `Escape` || event.key === `Tab`) {
         setOptionsVisible(false);
         searchText = ``;
+        input?.focus();
+    }
+    else if (event.key === 'ArrowDown' && !activeOption && !showOptions) {
+        // Open options on pressing down arrow key
+        setOptionsVisible(true)
     }
     // on enter key: toggle active option and reset search text
     else if (event.key === `Enter`) {
@@ -244,20 +251,24 @@ const if_enter_or_space = (handler) => (event) => {
   aria-multiselectable={maxSelect === null || maxSelect > 1}
   class:invalid
   class="multiselect {outerDivClass}"
-  on:mouseup|stopPropagation={() => setOptionsVisible(true)}
+  
   title={disabled ? disabledTitle : null}
   aria-disabled={disabled ? `true` : null}
 >
   <input
+    name="selectInput"
     {required}
     bind:value={formValue}
     tabindex="-1"
     aria-hidden="true"
-    aria-label="ignore this, used only to prevent form submission if select is required but empty"
+    aria-label="select input"
     class="form-control"
     on:invalid={() => (invalid = true)}
   />
-  <ExpandIcon width="15px" style="min-width: 1em; padding: 0 1pt;" />
+  <button type="button" name="expand options" class="expandButton inline pr-2 my-2 !z-50" on:mouseup|self|stopPropagation={() => setOptionsVisible(true)}
+    on:focus={() => setOptionsVisible(true)} >
+    <ExpandIcon width="16" height="16" class="w-5 h-5 p-1 m-0"/>
+  </button>
   <ul class="selected {ulSelectedClass}">
     {#each selected as option, idx}
       <li class={liSelectedClass} aria-selected="true">
@@ -269,6 +280,7 @@ const if_enter_or_space = (handler) => (event) => {
             on:mouseup|stopPropagation={() => remove(get_label(option))}
             on:keydown={if_enter_or_space(() => remove(get_label(option)))}
             type="button"
+            class="removeOptionButton"
             title="{removeBtnTitle} {get_label(option)}"
           >
             <CrossIcon width="15px" />
@@ -282,9 +294,7 @@ const if_enter_or_space = (handler) => (event) => {
         bind:this={input}
         {autocomplete}
         bind:value={searchText}
-        on:mouseup|self|stopPropagation={() => setOptionsVisible(true)}
         on:keydown={handleKeydown}
-        on:focus={() => setOptionsVisible(true)}
         {id}
         {name}
         {disabled}
@@ -338,6 +348,7 @@ const if_enter_or_space = (handler) => (event) => {
         on:mousedown|stopPropagation
         on:mouseup|stopPropagation={() => {
           if (!disabled) isSelected(label) ? remove(label) : add(label)
+          setOptionsVisible(false)
         }}
         title={disabled ? disabledTitle : (isSelected(label) && selectedTitle) || title}
         class:selected={isSelected(label)}
@@ -374,7 +385,7 @@ const if_enter_or_space = (handler) => (event) => {
           {addOptionMsg}
         </li>
       {:else}
-        <span>{noOptionsMsg}</span>
+          <li class="noOptionsMessage">{noOptionsMsg}</li>
       {/if}
     {/each}
   </ul>
@@ -417,18 +428,11 @@ const if_enter_or_space = (handler) => (event) => {
     flex-wrap: wrap;
   }
   :where(div.multiselect > ul.selected > li) {
-    align-items: center;
-    border-radius: 3pt;
-    display: flex;
-    margin: 2pt;
     line-height: normal;
     transition: 0.3s;
     white-space: nowrap;
-    background: var(--sms-selected-bg, rgba(0, 0, 0, 0.15));
-    padding: var(--sms-selected-li-padding, 1pt 5pt);
-    color: var(--sms-selected-text-color, var(--sms-text-color));
   }
-  :where(div.multiselect button) {
+  :where(div.multiselect button):not(.expandButton) {
     border-radius: 50%;
     display: flex;
     transition: 0.2s;
@@ -453,7 +457,7 @@ const if_enter_or_space = (handler) => (event) => {
   }
   :where(div.multiselect > ul.selected > li > input) {
     border: none;
-    outline: none;
+    outline: none !important;
     background: none;
     flex: 1; /* this + next line fix issue #12 https://git.io/JiDe3 */
     min-width: 2em;
@@ -461,6 +465,7 @@ const if_enter_or_space = (handler) => (event) => {
     font-size: inherit;
     cursor: inherit; /* needed for disabled state */
   }
+
   :where(div.multiselect > ul.selected > li > input)::placeholder {
     color: var(--sms-placeholder-color);
   }
@@ -511,11 +516,23 @@ const if_enter_or_space = (handler) => (event) => {
     color: var(--sms-li-selected-color);
   }
   :where(div.multiselect > ul.options > li.active) {
-    background: var(--sms-li-active-bg, var(--sms-active-color, rgba(0, 0, 0, 0.15)));
+    background: rgb(249 250 251);
   }
   :where(div.multiselect > ul.options > li.disabled) {
     cursor: not-allowed;
     background: var(--sms-li-disabled-bg, #f5f5f6);
     color: var(--sms-li-disabled-text, #b8b8b8);
   }
+
+  /* Custom CSS */
+  .removeOptionButton {
+    color: rgb(107 114 128);
+  }
+  .removeOptionButton:hover {
+    color: rgb(55 65 81);
+  }
+
+  .noOptionsMessage {
+    cursor: auto;
+  }
 </style>
diff --git a/node_modules/svelte-multiselect/icons/ChevronExpand.svelte b/node_modules/svelte-multiselect/icons/ChevronExpand.svelte
index 658985c..080279a 100644
--- a/node_modules/svelte-multiselect/icons/ChevronExpand.svelte
+++ b/node_modules/svelte-multiselect/icons/ChevronExpand.svelte
@@ -1,5 +1,5 @@
-<svg {...$$props} fill="currentColor" viewBox="0 0 16 16">
-  <path
-    d="M3.646 9.146a.5.5 0 0 1 .708 0L8 12.793l3.646-3.647a.5.5 0 0 1 .708.708l-4 4a.5.5 0 0 1-.708 0l-4-4a.5.5 0 0 1 0-.708zm0-2.292a.5.5 0 0 0 .708 0L8 3.207l3.646 3.647a.5.5 0 0 0 .708-.708l-4-4a.5.5 0 0 0-.708 0l-4 4a.5.5 0 0 0 0 .708z"
-  />
-</svg>
+  <svg {...$$props} fill="currentColor" viewBox="0 0 16 16">
+    <path
+      d="M3.646 9.146a.5.5 0 0 1 .708 0L8 12.793l3.646-3.647a.5.5 0 0 1 .708.708l-4 4a.5.5 0 0 1-.708 0l-4-4a.5.5 0 0 1 0-.708zm0-2.292a.5.5 0 0 0 .708 0L8 3.207l3.646 3.647a.5.5 0 0 0 .708-.708l-4-4a.5.5 0 0 0-.708 0l-4 4a.5.5 0 0 0 0 .708z"
+    />
+  </svg>
